Prüfschritte/de/11.2.4.4 Aussagekräftige Linktexte.adoc

```diff
@@ -4,102 +4,76 @@ include::include/attributes.adoc[]
 
-Ziel oder Zweck des Links sollen aus dem Linktext hervorgehen oder aus dem
-direkten Kontext des Links ermittelbar sein.
+Ziel oder Zweck des Links sollen aus dem Linktext hervorgehen oder aus dem direkten Kontext des Links ermittelbar sein.
 
-Falls Links nicht auf andere Ansichten der App verweisen, soll der Link über
-das Dateiformat des Zieldokuments informieren.
+Falls der Link nicht auf andere App-Ansichten verweist, soll der Link über das Dateiformat des Zieldokuments informieren.
 
-Wenn Links in Anwendungen nicht erwartbar auf Ressourcen außerhalb der
-Software verweisen, sollte der Linktext auch hier einen zusätzlichen Hinweis
-auf das Ziel(format) geben.
-Z. B. „öffnet in Browser“ etc.
+Auch wenn Links nicht auf Dokumente verweisen, sondern andere Anwendungen öffnen, sollte der Linktext auch hier einen zusätzlichen Hinweis auf das Ziel gebenb Z. B. „öffnet im Browser“ etc.
 
 == Warum wird das geprüft?
 
-Blinde Nutzer, die von Link zu Link tabben / navigieren, bekommen die
-Linktexte vorgelesen und können bei aussagekräftigen Linktexten leicht
-entscheiden, ob sie einem Link folgen möchten.
-
-Falls der Linktext selbst nicht aussagekräftig ist, soll der unmittelbare
-Kontext für Screenreader-Nutzer wenigstens leicht ermittelbar sein.
-
-Screenreader bieten die Möglichkeit der Auflistung bzw.
-direkter Anspringbarkeit sämtlicher Links der App-Ansicht und damit auch
-einen schnellen Überblick, selbst wenn die Software ansonsten schlecht
-zugänglich ist.
-Diese Technik funktioniert allerdings nicht, wenn alle Linktexte gleich sind
-und nicht ausreichend Auskunft über das Linkziel geben.
-
-Für Angebote außerhalb der App (zum Beispiel PDF- oder Word-Dokumente)
-benötigt der Benutzer zusätzliche Apps oder Plugins, die aber nicht jeder
-installiert hat.
-Manche Apps arbeiten schlecht mit Screenreadern oder anderen Hilfsanwendungen
-zusammen, manche Dateiformate sind nicht oder nur schlecht zugänglich.
-Für viele Benutzer ist es wichtig zu wissen, in welchem Format
-Informationen angeboten werden, bevor sie einen Link aktivieren.
-
-Ein weiterer Grund, weshalb die Vorab-Information über den Typ des Links
-wichtig ist: Der Benutzer weiß dann, was auf ihn zukommt.
-Er ist nicht überrascht oder irritiert, wenn plötzlich Funktionen nicht mehr
-vorhanden sind oder der E-Mail-Client geöffnet wird.
+Blinde Nutzende, die von Link zu Link navigieren, bekommen die Linktexte vorgelesen und können bei aussagekräftigen Linktexten leicht entscheiden, ob sie einem Link folgen möchten.
+
+Falls der Linktext selbst nicht aussagekräftig ist, soll der unmittelbare Kontext für Screenreader-Nutzende wenigstens leicht ermittelbar sein.
+
+Screenreader bieten die Möglichkeit sämtliche Links der Seite aufzulisten und ermöglichen damit
+einen schnellen Überblick, selbst wenn die Software ansonsten schlecht zugänglich ist. Diese Technik funktioniert allerdings nicht, wenn alle Linktexte gleich sind und nicht ausreichend Auskunft über das Linkziel geben.
```

Comment:
Ein Auflistung wie in Desktop-Screenreadern gibt es meines Wissens nicht bei iOAS und Android. Man kann über den Rotor gezielt nur die Links durchlaufen.
Man kann das aber umformulieren:
"Screenreader bieten die Möglichkeit, gezielt nur bestimmte Elemente, wie z.B. alle Links auf der Ansicht, zu durchlaufen, und ermöglichen damit einen besseren Überblick. Bei bei mehreren gleichen Linktexten wie "Weiter" oder "mehr lesen" ist das Linkziel schwerer zu ermitteln."

Reply:
Ich habe den Absatz durch deine vorgeschlagene Formulierung "Screenreader bieten die Möglichkeit..." ersetzt.

In diesem, wie auch bei Prüfschritten wie auch bei einigen anderen Prüfschritten frage ich mich, ob beim Testen
Links und Schaltern und deren semantisch korrekter Einsatz unterschieden wird. D.h. bemängelt man hier die Aussagekraft von Links und bei 2.4.6 Schalter? Und bemängelt man bei 4.1.2 den falschen Einsatz von Schaltern / Links? 
Wann sollte bei Apps ein Schalter, wann ein Link ausgegeben werden? Neue Ansicht = Link, sonst Schalter?